x/concentrated-liquidity/tick.go

```diff
@@ -81,15 +81,23 @@ func (k Keeper) SetTickInfo(ctx sdk.Context, poolId uint64, tickIndex int64, tic
 // Also, lower tick must be less than upper tick.
 // Returns error if validation fails. Otherwise, nil.
 // TODO: test
-func validateTickRangeIsValid(lowerTick int64, upperTick int64) error {
-	// ensure types.MinTick <= lowerTick < types.MaxTick
+func validateTickRangeIsValid(tickSpacing uint64, lowerTick int64, upperTick int64) error {
```

Comment:
Let's add tests to this function, please. If none exist, let's make an issue to test it